ansible/cluster.yml

```diff
@@ -65,6 +71,7 @@
 # install kubernetes on the nodes
 - hosts: nodes
   sudo: yes
+  serial: 5
```

Comment:
Is this required for adding contiv support? If not, can you please create a separate pr for the patch?